### ISSUE TITLE
Does not work after latest update

### ISSUE THREAD

**Issue by LuisA92** [AckslD/swenv.nvim#8]
My install stopped working with the latest update. 

I am able to get it to work be removing the following lines: 

```vim
local conda_env_path = Path:new(vim.fn.getenv('CONDA_EXE')):parent():parent() .. '/envs'
local conda_paths = scan_dir(conda_env_path, { depth = 1, only_dirs = true, silent = true })

for _, path in ipairs(conda_paths) do
	table.insert(venvs, {
		name = Path:new(path):make_relative(conda_env_path),
		path = path,
		source = 'conda',
	})
end
```

Is there anyway to add support for [micromamba](https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html)?

**Comment by AckslD** [comment]
Thanks for the issue, should be fixed now by #9 

Re `micromamba`, I won't myself but feel free to give it a try. Could follow the same pattern as venv or conda although I have no idea how micromamba works

**Comment by LuisA92** [comment]
I got it working with a minor change for getting the environment path: 

```lua
  local micromamba_exe = vim.fn.getenv('MAMBA_EXE')
  if micromamba_exe ~= vim.NIL then
    local micromamba_env_path = Path:new(vim.fn.getenv('MAMBA_ROOT_PREFIX')) .. '/envs'
    local micromamba_paths = scan_dir(micromamba_env_path, { depth = 1, only_dirs = true, silent = true })

    for _, path in ipairs(micromamba_paths) do
      table.insert(venvs, {
        name = Path:new(path):make_relative(micromamba_env_path),
        path = path,
        source = 'micromamba',
      })
    end
  end
```
If you'd like to add micromamba support its in PR [#10](https://github.com/AckslD/swenv.nvim/pull/10)
